contracts/src/v0.8/ccip/test/pools/GHO/GhoTokenPoolRemote.t.sol

```diff
@@ -0,0 +1,503 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {GhoToken} from "@aave-gho-core/gho/GhoToken.sol";
+import {TransparentUpgradeableProxy} from "solidity-utils/contracts/transparent-proxy/TransparentUpgradeableProxy.sol";
+
+import {stdError} from "forge-std/Test.sol";
+import {UpgradeableTokenPool} from "../../../pools/GHO/UpgradeableTokenPool.sol";
+import {EVM2EVMOnRamp} from "../../../onRamp/EVM2EVMOnRamp.sol";
+import {EVM2EVMOffRamp} from "../../../offRamp/EVM2EVMOffRamp.sol";
+import {BurnMintTokenPool} from "../../../pools/BurnMintTokenPool.sol";
+import {RateLimiter} from "../../../libraries/RateLimiter.sol";
+import {Pool} from "../../../libraries/Pool.sol";
+import {MockUpgradeable} from "../../mocks/MockUpgradeable.sol";
+
+import {GhoTokenPoolRemoteSetup} from "./GhoTokenPoolRemoteSetup.t.sol";
+
+contract GhoTokenPoolRemote_lockOrBurn is GhoTokenPoolRemoteSetup {
+  function testSetupSuccess() public view {
+    assertEq(address(s_burnMintERC677), address(s_pool.getToken()));
+    assertEq(address(s_mockRMN), s_pool.getRmnProxy());
+    assertEq(false, s_pool.getAllowListEnabled());
+    assertEq("BurnMintTokenPool 1.5.1", s_pool.typeAndVersion());
+  }
+
+  function testPoolBurnSuccess() public {
+    uint256 burnAmount = 20_000e18;
+    // inflate facilitator level
+    _inflateFacilitatorLevel(address(s_pool), address(s_burnMintERC677), burnAmount);
+
+    deal(address(s_burnMintERC677), address(s_pool), burnAmount);
+    assertEq(s_burnMintERC677.balanceOf(address(s_pool)), burnAmount);
+
+    vm.startPrank(s_burnMintOnRamp);
+
+    vm.expectEmit();
+    emit TokensConsumed(burnAmount);
+
+    vm.expectEmit();
+    emit Transfer(address(s_pool), address(0), burnAmount);
+
+    vm.expectEmit();
+    emit Burned(address(s_burnMintOnRamp), burnAmount);
+
+    bytes4 expectedSignature = bytes4(keccak256("burn(uint256)"));
+    vm.expectCall(address(s_burnMintERC677), abi.encodeWithSelector(expectedSignature, burnAmount));
+
+    (uint256 preCapacity, uint256 preLevel) = GhoToken(address(s_burnMintERC677)).getFacilitatorBucket(address(s_pool));
+
+    s_pool.lockOrBurn(
+      Pool.LockOrBurnInV1({
+        originalSender: OWNER,
+        receiver: bytes(""),
+        amount: burnAmount,
+        remoteChainSelector: DEST_CHAIN_SELECTOR,
+        localToken: address(s_burnMintERC677)
+      })
+      // OWNER, bytes(""), burnAmount, DEST_CHAIN_SELECTOR, bytes("")
```

Comment:
This looks like the old pool's parameters, right? I think safe to remove this comment now

Reply:
yep, left as reference for the review. can remove!